hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -90,7 +90,7 @@ private void handleGetClusterVersion(HttpGetCommand command) {
 
     private void handleCluster(HttpGetCommand command) {
         Node node = textCommandService.getNode();
-        StringBuilder res = new StringBuilder(node.getClusterService().membersString());
+        StringBuilder res = new StringBuilder(node.getClusterService().memberListString());
```

Comment:
You're right. Member list format switch should affect the rest response too. will fix that.